external_auth/views.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.shortcuts import render
+
+from django.contrib import messages
+from external_auth.forms import PakwheelsLoginForm
+from external_auth.pakwheels_scraper import PakwheelsScraper
+from external_auth.pakwheels_profile import PakwheelsProfile
+
+
+def pakwheels_registration(request):
+
+    if request.method == 'POST':
```

Comment:
instead of checking `POST` first I would suggest returning `GET` response first. That way the code will become more readable and the indentation of `POST` code will become one step back.
```python
if request.method == 'GET':
    form = PakwheelsLoginForm()
    context = {'form': form}
    return render(request, 'pakwheels_register.html', context)

# POST request handler
form = PakwheelsLoginForm(request.POST)
...
```
Also allow `GET` and `POST` method for this view: https://docs.djangoproject.com/en/1.11/topics/http/decorators/#allowed-http-methods